lib/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var watcherServerMiddleware = require('./middleware');
+var serveAssetMiddleware = require('./serve-asset-middleware');
+var watcherMiddleware = require('./watcher-middleware');
+
+module.exports = {
+  'watcherServerMiddleware': watcherServerMiddleware,
```

Comment:
Yeah I am fine any path. I don't have much experience on how it works in ember-cli.